### PR TITLE
Upgrade dependencies for security fix

### PR DESCRIPTION
**Pull request by JosephJang7227**

### CODE REVIEW

**Review by leqiang-deltafi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -15,25 +15,38 @@
...
+    "@pythnetwork/client": "^2.5.1",
+    "@reduxjs/toolkit": "^1.6.1",
     "@solana/spl-token": "^0.1.8",
     "@solana/spl-token-registry": "^0.2.303",
+    "@solana/web3.js": "^1.43.2",
     "@testing-library/jest-dom": "^5.16.2",
-    "@pythnetwork/client": "^2.5.1",
-    "@project-serum/anchor": "~0.24.2",
-    "@reduxjs/toolkit": "^1.6.1",
+    "ansi-html": "0.0.8",
```
Comment (ypeng-deltafi) [inline package.json:25]: can you use `^0.0.8`?

Comment (ypeng-deltafi) [reply package.json:25]: is there any specific reason to use fixed version instead of using `^` to allow higher version?

Comment (ypeng-deltafi) [reply package.json]: can you also change other dependencies?

Comment (JosephJang7227) [reply package.json]: For every dependency without "^"?

**Review by ypeng-deltafi — APPROVED** [review]

**Pull request merged**